app/views/home/index.html.erb

```diff
@@ -18,22 +18,26 @@
     <h1>みんなのポエム一覧</h1>
   </div>
   <p>
-    <%= link_to "ポエムを作成する", new_poem_path, class: "btn btn-primary btn-lg" %>
+  <%= link_to "ポエムを作成する", new_poem_path, class: "btn btn-primary btn-lg" %>
   </p>
   <ul class="list-group">
     <% @poems.each do |poem| %>
-    <li class="list-group-item pull-left col-md-4">
-      <div class="icon-area">
-        <%= image_tag(poem.user.image_url, class: "img-circle", width: 60) %>
-      </div>
-      <div>
-        <%= link_to poem.title.truncate(20), poem %>
-      </div>
-      <div>
-        <%= poem.user.name %>
-      </div>
-      <div class="text-right">
-        <%= poem.created_at %>
+      <li class="list-group-item col-md-4">
+        <div class="medila">
+          <div class="media-left">
+            <%= image_tag(poem.user.image_url, class: "img-circle", width: 60) %>
+          </div>
+          <div class="media-body">
+            <div class="media-heading">
+              <%= link_to poem.title.truncate(15), poem %>
+            </div>
+            <div class="media-content">
+              <%= poem.user.name %>
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)